src/resolvers/dmsp.ts

```diff
@@ -1,13 +1,26 @@
+import { formatLogMessage } from '../logger';
 import { Resolvers } from "../types";
 
+function prepareLogger(logger, session = null) {
+
+}
+
 export const resolvers: Resolvers = {
   Query: {
     // returns a DMSP that matches the specified DMP ID
-    dmspById: (_, { dmspId }, { dataSources }) => {
+    dmspById: (_, { dmspId }, { logger, dataSources }) => {
+      const logMessage = `Resolving query dmspById(dmspId: '${dmspId}')`;
+
       return new Promise((resolve, reject) => {
         dataSources.dmphubAPIDataSource.getDMSP(encodeURIComponent(dmspId))
-          .then(rows => resolve(rows))
-          .catch(error => reject(error));
+          .then(rows => {
+            formatLogMessage(logger).debug(logMessage);
+            resolve(rows)
+          })
+          .catch(err => {
+            formatLogMessage(logger, { err, dmspId }).debug(`ERROR: ${logMessage} - ${err.message}`);
```

Comment:
@briri  - Line 21 - If you are logging this as an "ERROR", would it make more sense to use ".error()" instead of ".debug()" on this?

Reply:
yep. 💯